### PR TITLE
Adding Microsoft.Storage 2017-06-01

### PR DESCRIPTION
**Pull request by JasonYang-MSFT**

### CODE REVIEW

File: schemas/2017-06-01/storageAccounts.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
+# Microsoft.Storage/storageAccounts template reference
```
Comment (TianoMS) [inline schemas/2017-06-01/storageAccounts.md]: >Microsoft.Storage/storageAccounts template reference [](start = 2, length = 52)

we don't support .md files in this repo.

File: schemas/2017-06-01/Microsoft.Storage.json [hunk 2]
```diff
@@ -0,0 +1,526 @@
...
+          ],
+          "description": "The parameters used to create the storage account."
+        }
+      },
+      "required": [
+        "name",
+        "type",
+        "apiVersion",
+        "sku",
+        "kind",
```
Comment (TianoMS) [inline schemas/2017-06-01/Microsoft.Storage.json]: Are these required?

Comment (JasonYang-MSFT) [reply schemas/2017-06-01/Microsoft.Storage.json]: removed.

File: schemas/2017-06-01/storageAccounts.md (hunk 1 above)
Comment (JasonYang-MSFT) [reply schemas/2017-06-01/storageAccounts.md]: removed

**Pull request merged**